cmd/build.go

```diff
@@ -83,6 +84,7 @@ func newBuildCmd(out io.Writer) *cobra.Command {
 
 	f.StringVarP(&r.registryUser, "username", "u", "", "the username to use when logging into the registry")
 	f.StringVarP(&r.registryPw, "password", "p", "", "the password to use when logging into the registry")
+	f.StringArrayVar(&r.credentials, "credentials", []string{}, "all credentials for private repos")
```

Comment:
`run` needs this as well; make a few functions to share between them for the split and append